examples/s2-parcel-example/src/App.js

```diff
@@ -223,6 +301,26 @@ function App() {
         </Section>
 
         <Section title="Navigation">
+          <div className={style({ minHeight: 176 })}>
+            <Accordion>
+              <Disclosure id="files">
+                <DisclosureHeader>
```

Comment:
these need to be updated to be `<DisclosureTitle>`

also might be good to have an example of a disclosure with an action button

Reply:
Ahh yeah, thanks!